sycl/test-e2e/KernelFusion/lit.local.cfg

```diff
@@ -2,7 +2,7 @@ import platform
 
 config.required_features += ['fusion']
 # TODO: Reenable hip, see https://github.com/intel/llvm/issues/14598
```

Comment:
Shouldn't this comment be dropped? Also, the description might make use of something like "Closes <issue-link>"

Reply:
yes, fixed, thanks